pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -255,7 +258,10 @@ func configFromEnv() (cfg Config, ok bool) {
 	return
 }
 
+// ReadConfig reads values from environment variables and the cloud.conf, prioritizing cloud-config
+// WARNING: depricated
 func ReadConfig(config io.Reader) (Config, error) {
+	klog.Warning("Use of cloud-configs are depricated, and may no longer be supported in the future")
```

Comment:
I think i am going to move this to an else statement at line 286 instead. The current implementation makes no sense